File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int counter = 1;
        for(int i = 1; i <nums.size(); i++){
            if(nums[i] != nums[i-1]){
                nums[counter] = nums[i];
                counter++;
            }
        }
        return counter;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        s = set()
        i , j = 0, 0
        while i < n:
            if nums[i] not in s:
                nums[j] = nums[i]
                s.add(nums[i])
                j += 1
            i += 1
        return j

        